### DISCUSSION TITLE
How to draw molecule in other way?

### DISCUSSION THREAD

**Discussion by Hanbit-Kim-0515** [rdkit/rdkit#5040] [Q&A]
I want to draw molecule in various fashion for a kind of augmentation
So I put many options as below code.
However, what I want most of all is to draw molecule in uneven handwriting or casual manner.
How can I draw in these ways?
Or is there any option you recommend for augmentation of drawing molecule images ? 

-----------code------------
options = MolDrawOptions()
options.useBWAtomPalette()
options.rotate = round(random.uniform(0, 360),0)
options.bondLineWidth = 8
options.atomLabelFontSize = 1
options.minFontSize = int(round(random.uniform(6, 18),0))
options.additionalAtomLabelPadding = round(random.uniform(0, 0.2),1)
options.padding = round(random.uniform(0.04, 0.06), 2)
options.explicitMethyl = bool(random.getrandbits(1))

image = Chem.Draw.MolToImage(mol, size=(1200,1200), options = options)
image
-----------code-------------

**Comment by sunhwan** [comment]
Are you referring to the "XKCD-style" drawing? You can turn it on by set `comicMode` as `True`. 

> from IPython.display import SVG
> mol = Chem.MolFromSmiles('CC1=C(C=C(C=C1)NC(=O)C2=CC=C(C=C2)CN3CCN(CC3)C)NC4=NC=CC(=N4)C5=CN=CC=C5')
> d2d = rdMolDraw2D.MolDraw2DSVG(350,300)
> options = d2d.drawOptions()
> options.comicMode = True
> options.fontFile = '/System/Library/Fonts/Supplemental/Comic Sans MS Bold.ttf' # this is from my Mac OS version 12.1
> d2d.DrawMolecule(mol)
>d2d.FinishDrawing()
> SVG(d2d.GetDrawingText())

<img width="334" alt="Screen Shot 2022-02-23 at 12 06 44 AM" src="https://user-images.githubusercontent.com/94840/155281304-12d2da24-76a9-417d-b0e2-5f2da5ecf462.png">

**Reply by Hanbit-Kim-0515** [reply]
This is exactly what I wanted. 
Thank you so much:)
